src/Bridge/Symfony/Bundle/Resources/config/api.xml

```diff
@@ -61,6 +61,7 @@
         <service id="api_platform.serializer.context_builder" class="ApiPlatform\Core\Serializer\SerializerContextBuilder" public="false">
             <argument type="service" id="api_platform.metadata.resource.metadata_factory" />
         </service>
+        <service id="ApiPlatform\Core\Serializer\SerializerContextBuilder" alias="api_platform.serializer.context_builder.filter.inner"/>
```

Comment:
Shouldn't the alias name been the interface?

Reply:
👍

Reply:
Yeah you are right